reactioncommerce/reaction
# Inviting an existing Shop user as admin should promote that account to admin

Related to #2195

This is to update the "Add Admin User" form/process to handle inviting an existing shop user as admin. It should promote that account to admin. This will deprecate the "Add User To Group" section of the accounts dashboard UI.

To test:
_ Use the "Add Admin User" form on the upper right of Accounts dashboard
_ Invite an already existing user and confirm that the user is promoted to the new group

Changes introduced by this PR:
_ Updates "inviteShopMember" method to handle existing user emails. It previously throws a "user exists" message.
_ Now the "inviteShopMember" method called "groups/addUser" when it's given an existing user email, and then sends email
_ Also fixed a bug in getCurrentUserName method so it uses name of the user sending invite and not always"admin"
_ Removed AddGroupMembers component. The above replaces it.
